### PR TITLE
fix: Update models file

### PR DESCRIPTION
**Pull request by cyaiox**
This PR validates the model size, including the thumbnail size, until there's a clear definition of how the catalyst will calculate it.

### CODE REVIEW

File: src/modules/item/utils.ts [hunk 1]
```diff
@@ -756,3 +762,19 @@ export const buildItemMappings = (mapping: Mapping, contract: LinkedContract): M
     }
   }
 }
+
+export const isWearableFileSizeValid = (fileSize: number): boolean => {
+  return fileSize > MAX_WEARABLE_FILE_SIZE
```
Comment (meelrossi) [inline src/modules/item/utils.ts:767]: shouldn't this be `<`? 

Maybe change the name to invalid? I see that you are using it that if it returns true then its invalid

Comment (cyaiox) [reply src/modules/item/utils.ts]: you're right! fixing it

Comment (cyaiox) [reply src/modules/item/utils.ts]: fixed 🙌

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx [hunk 2]
```diff
@@ -1085,7 +1091,7 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
       return false
     }
 
-    if (isRequirementMet && !isSkin && !isSmartWearable && modelSize && modelSize > MAX_WEARABLE_FILE_SIZE) {
+    if (isRequirementMet && !isSkin && !isSmartWearable && finalSize && !isWearableFileSizeValid(finalSize)) {
```
Comment (meelrossi) [inline src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx:1094]: are we missing the !isEmote here?
```suggestion
    if (isRequirementMet && !isSkin && !isSmartWearable && !isEmote && finalSize && !isWearableFileSizeValid(finalSize)) {
```

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx [hunk 3]
```diff
@@ -1075,7 +1081,7 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
       return false
     }
 
-    if (isRequirementMet && !isSkin && isSmartWearable && modelSize && modelSize > MAX_SMART_WEARABLE_FILE_SIZE) {
+    if (isRequirementMet && !isSkin && isSmartWearable && finalSize && !isSmartWearableFileSizeValid(finalSize)) {
```
Comment (meelrossi) [inline src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx:1084]: can it be a smart wearable and skin?

**Review by meelrossi — APPROVED** [review]

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx (hunk 3 above)
Comment (cyaiox) [reply src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx:1084]: mmmm, it's a good point. the smart wearables can be skin but have the limitation for the `js` files of 3MB max. 

will update the condition to have a better validation

**Pull request merged**